jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -45,142 +40,40 @@ class Status extends React.Component {
   componentWillReceiveProps() {
```

Comment:
This function is now deprecated in React, I feel like this would be a good time to update it.

https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops